website/src/pages/contribute/ContributeCard.js

```diff
@@ -0,0 +1,120 @@
+  color: ${(props) => (props.isShow ? "rgb(232, 67, 62)" : "")};
+`;
+
+const Issue = styled.div`
+  @media (max-width: 700px) {
+    min-width: 5rem;
+  }
+  border: 1px solid rgb(232, 67, 62);
+  border-radius: 0.5rem;
+  padding: 0.25rem 0.5rem;
+  font-size: .875rem;
+`;
+
+const ProjectDesc = styled.div`
+  display: flex;
+  color: ${(props) => (props.isShow ? "rgb(232, 67, 62)" : "")};
+`;
+const List = styled.div`
+    display: ${(props) => (props.isShow ? "block" : "none")};
+`;
+const ListItem = styled.li`
+  list-style: none;
+  display: flex;
+`;
+
+
+const ContributeCard = (props) => {
+  const { repoName } = props;
+  const [isShow, setisShow] = useState(false);
+  const [repoInfo, setRepoInfo] = useState({ description: '', Star: '', Watch: '', Fork: '' });
+  const [issues, setIssues] = useState([]);
+
+  useEffect(() => {
+    getGitHubIssuesInfo(repoName).then((result) => {
+      setIssues(result);
+    });
+    getGitHubRepoInfo(repoName).then((result) => {
+      setRepoInfo({ description: result.description, Star: result.stargazers_count, Watch: result.subscribers_count, Fork: result.forks_count })
+    })
+  }, []);
+  return (
+    <Card onClick={() => setisShow(!isShow)} isShow={isShow}>
+      <ProjectTitle>
+        <Title isShow={isShow}>{repoName}</Title>
+        <Issue isShow={isShow}>{issues.length} issues</Issue>
+      </ProjectTitle>
+      <div>{repoInfo.description}</div>
+      <ProjectDesc isShow={isShow}>
+        <div style={{ marginRight: '1rem' }}>Star: {repoInfo.Star}</div>
+        <div style={{ marginRight: '1rem' }}>Watch: {repoInfo.Watch}</div>
+        <div style={{ marginRight: '1rem' }}>Fork: {repoInfo.Fork}</div>
+      </ProjectDesc>
+      <List isShow={isShow}>
+        <ul style={{ paddingLeft: 0 }}>
+          {issues.map(item => (
+            <ListItem key={item.number}>
+              <div style={{ minWidth: '4rem' }}>#{item.number}</div>
+              <a target="_blank" href={item.html_url} style={{ flex: '1 1 auto', textDecoration: 'none', overflow: 'hidden' }}>{item.title} </a>
+              {item.comments > 0 ? <div style={{ display: "flex", justifyContent: 'center', alignItems: 'center' }}><IconComment></IconComment><div style={{ marginLeft: '0.25rem', fontSize: '0.5rem', color: '#333' }}>{item.comments}</div></div> : ''}
+            </ListItem>
+          ))}
+        </ul></List>
+    </Card>
+  );
+};
+
+const getGitHubIssuesInfo = (repo) => {
+  return fetch(`https://api.github.com/repos/${repo}/issues?state=open&labels=good%20first%20issue`, {
+    headers: {
+      "content-type": "application/json",
+      Accept: "application / vnd.github.v3 + json",
+    },
+  }).then((response) => response.json()
+  );
+};
+
+const getGitHubRepoInfo = (repo) => {
+  return fetch(`https://api.github.com/repos/${repo}`, {
+    headers: {
+      "content-type": "application/json",
+      Accept: "application / vnd.github.v3 + json",
+    },
+  }).then((response) => response.json());
+};
```

Comment:
GitHub API has frequency now, while some regions are not able to access the host properly, can we move it to Build CI for that?